apps/sms/views/shared/test/unit/settings_test.js

```diff
@@ -36,8 +36,12 @@ suite('Settings >', function() {
     Settings.maxConcatenatedMessages = 10;
   });
 
-  test('getSimNameByIccId returns the empty string before init', function() {
-    assert.equal('', Settings.getSimNameByIccId('anything'));
+  test('getSimNameByIccId returns the empty string before init',
+    function(done) {
+      Settings.getSimNameByIccId('anything').then(function(ret) {
+        assert.equal('', ret);
+        done();
+      });
```

Comment:
nit: use the following construct:
```javascript
functionReturningPromise.then(() => {
  ... assert stuff
}).then(done, done);
```

Then "done" correctly catch any assertion error and forward it to mocha; otherwise you lose it and it just times out